tests/tests.go

```diff
@@ -209,6 +210,26 @@ func initKubemacpoolParams(rangeStart, rangeEnd string) error {
 	return nil
 }
 
+func getVmFailCleanupWaitTime() (time.Duration, error) {
+	managerDeployment, err := testClient.KubeClient.AppsV1().Deployments(managerNamespace).Get(names.MANAGER_DEPLOYMENT, metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred(), "Should successfully get Deployment object")
+	Expect(len(managerDeployment.Spec.Template.Spec.Containers) != 0).To(BeTrue(), "Container array size should be greater then zero")
+
+	for _, container := range managerDeployment.Spec.Template.Spec.Containers {
+		if container.Name == deploymentContainerName {
+			for _, arg := range container.Args {
+				waitTime := 0
+				if elementsParsed, err := fmt.Sscanf(arg, "--wait-time=%d", &waitTime); elementsParsed == 1 {
```

Comment:
Let's reduce a little the identation level, if there is an error we don't care for elementsParsed
```golang
elementsParsed, err := fmt.Sscanf(arg, "--wait-time=%d", &waitTime)
Expect(err).ToNot(HaveOccurred(), "Should successfully parse wait-time arg")
if elementsParsed == 1 {
    return time.Duration(waitTime) * time.Second
}
```

Reply:
I ca't use Expect(err), we have more than 1 argument, and I want it to run through all the arguments and return only once it found the wait-time arg.
I can do something like 
```
			for _, arg := range container.Args {
				waitTime := 0
				elementsParsed, err := fmt.Sscanf(arg, "--wait-time=%d", &waitTime)
				if err != nil {
					continue
				}
				if elementsParsed == 1 {
					Expect(err).ToNot(HaveOccurred(), "Should successfully parse wait-time arg")
					return time.Duration(waitTime) * time.Second
				}
```
what do you think?

Reply:
Sscanf does not fail if does not find it if it fails is because the string is wrong or the like so we should stop the test.

Reply:
ack